frontend/src/Components/Utilities/DetailView/CaseList.tsx

```diff
@@ -53,6 +52,4 @@ interface CaseItemProps {
 
 const CaseItem = styled(ListItem) <CaseItemProps>`
   background:${props => props.isSelected ? "#ecbe8d" : 'none'};
-`;
-
-
+`;
```

Comment:
Probably could use a newline on the end of this file

Reply:
Made the fix on the public-state branch (oops I forgot to switch branches before push the change...but anyway this shouldn't be an issue)